redhat/src/stack/images/7.6.1810/initrd.img/opt/stack/bin/stacki-profile.py

```diff
@@ -0,0 +1,105 @@
+#!/opt/stack/bin/python3 -E
+#
+# @copyright@
+# Copyright (c) 2006 - 2019 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+#
+
+import random
+import re
+import subprocess
+import time
+import sys
+
+
+# Start to build the curl command
+command = [
+	'/usr/bin/curl', '-s', '-w', '%{http_code}', '--local-port', '1-100',
```

Comment:
While we're in here, can we replace the shortopts?

```
-s, --silent
-w, --write-out <format>
```

Reply:
I hate you all

Reply:
<3